src/HealthChecks.MongoDb/DependencyInjection/MongoDbHealthCheckBuilderExtensions.cs

```diff
@@ -12,40 +12,14 @@ public static class MongoDbHealthCheckBuilderExtensions
     private const string NAME = "mongodb";
 
     /// <summary>
-    /// Add a health check for MongoDb database that list all databases on the system.
+    /// Add a health check for MongoDb that list all databases from specified <paramref name="clientFactory"/>.

```

Comment:
This summary is incorrect in some cases because if you pass a databaseName it will just ping that database.